content/blog/2018-02-27-sourcebots-2018-competition.md

```diff
@@ -0,0 +1,12 @@
+---
+title: SourceBots 2018 Competition
+date: 2018-02-27
+---
+
+Today's the day we announce the date of the competition!
+
+The competition will be held from Saturday 21st April until Sunday 22nd April, at the University of Southampton's Highfield campus, in the Cube. The exact start/end times are to be confirmed, but it'll start at some point in the morning of the 21st, and finish in the afternoon of the 22nd, no later than 5:00pm.
+
+Media consent will be required to attend the competition. The forms for this will be sent through in the next week or two.
```

Comment:
From what I've seen, the media consent details are being sent out ~now; we should update this to match.

Reply:
This blog post is backdated, so at time of publish, it's relevant

Reply:
It's back-dated to over a month ago; I don't think we should be back-dating that far. When the PR was first written the back-dating was a couple of days, which I think is ok. I think that back-dating more than a week will confuse people. (see also my other comment about back-dating)